XForm/XForm/XForm.csproj

```diff
@@ -11,6 +11,21 @@
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <PublishUrl>publish\</PublishUrl>
```

Comment:
Any idea what caused these changes? Could you back them out and just re-add Coalesce.cs?

If your version of VS ends up continually re-adding them, feel free to submit them.